libs/ledger-live-common/src/families/evm/api/etherscan.ts

```diff
@@ -109,16 +117,184 @@ export const getLastTokenOperations = makeLRUCache<
 
     return ops
       .map((event) => etherscanERC20EventToOperation(accountId, event))
-      .filter(Boolean) as {
-      tokenCurrency: TokenCurrency;
-      operation: Operation;
-    }[];
+      .filter(Boolean) as Operation[];
+  },
+  (currency, address, accountId, fromBlock) => accountId + fromBlock,
+  { ttl: 5 * 1000 }
+);
+
+/**
+ * Get all the last ERC721 transactions
+ */
+export const getLastERC721Operations = makeLRUCache<
```

Comment:
We could cover getLastERC721Operations, getLastERC1155Operations and getLastNftOperations with unit tests

Reply:
It's not only a network call, for example apiDomain is null, you don't call the network and return []